src/main/java/supersymmetry/common/metatileentities/multi/electric/MetaTileEntitySinteringOven.java

```diff
@@ -0,0 +1,122 @@
+package supersymmetry.common.metatileentities.multi.electric;
+
+import gregtech.api.capability.impl.MultiblockRecipeLogic;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.interfaces.IGregTechTileEntity;
+import gregtech.api.metatileentity.multiblock.IMultiblockPart;
+import gregtech.api.metatileentity.multiblock.RecipeMapMultiblockController;
+import gregtech.api.pattern.BlockPattern;
+import gregtech.api.pattern.FactoryBlockPattern;
+import gregtech.api.pattern.PatternMatchContext;
+import gregtech.api.recipes.Recipe;
+import gregtech.api.unification.material.Materials;
+import gregtech.api.util.GTUtility;
+import gregtech.client.renderer.ICubeRenderer;
+import gregtech.client.renderer.texture.Textures;
+import gregtech.common.blocks.BlockMachineCasing;
+import gregtech.common.blocks.MetaBlocks;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.Style;
+import net.minecraft.util.text.TextComponentTranslation;
+import net.minecraft.util.text.TextFormatting;
+import org.jetbrains.annotations.NotNull;
+import supersymmetry.api.block.BlockSinteringBrick;
+import supersymmetry.api.metatileentity.multiblock.SuSyPredicates;
+import supersymmetry.api.recipe.SuSyRecipeMaps;
+import supersymmetry.api.recipe.properties.SinterProperty;
+
+import java.util.List;
+
+public class MetaTileEntitySinteringOven extends RecipeMapMultiblockController {
+
+    private boolean canUsePlasma;
+
+    public MetaTileEntitySinteringOven(ResourceLocation metaTileEntityId) {
+        super(metaTileEntityId, SuSyRecipeMaps.SINTERING_RECIPES);
+        this.recipeMapWorkable = new MultiblockRecipeLogic(this);
+    }
+
+    @Override
+    public MetaTileEntity createMetaTileEntity(IGregTechTileEntity tileEntity) {
+        return new MetaTileEntitySinteringOven(metaTileEntityId);
+    }
+
+    @Override
+    protected void addDisplayText(List<ITextComponent> textList) {
+        if (isStructureFormed()) {
+            textList.add(new TextComponentTranslation("gregtech.multiblock.sintering_oven.can_use_plasma",
+                    new TextComponentTranslation(canUsePlasma ? "Yes" : "No")
```

Comment:
"Yes" and "No" should have their own translation keys. If you don't want to do this, you can change it to `new TextComponentString(...)`.